docs/details/api.md

```diff
@@ -19,3 +19,45 @@ The objects visible are nodes, services, and agents:
 Nodes, Services, and Agents have **Types** which define specific their properties, and their specific logic.
 
 Nodes and Services are external by nature – we do not manage them (create, destroy), but merely maintain a list of them (add to inventory, remove from inventory) in `pmm-managed`. Most Agents are started and stopped by `pmm-agent`. One exception is the External Exporter Type which is started externally.
+
+
+## API Keys and authentication
+
+API keys are used to control access to the PMM server components and resources. With an API key, you are authenticated to the PMM server, have access to PMM server components and resources, and perform various actions on them. You can use API keys as a replacement for basic authentication.
+
+### Generate API keys
+
+PMM uses the Grafana API keys for authentication. Following are the steps to generate the API keys:
+
+1. Login to PMM.
+2. From the side menu, click *Configuration  → API keys*.
+3. On the Configuration page, click *Add API Key*.
+4. *Add API key* dialog box opens.
+5. Enter  the following to generate an API key:
+    - key name (you can give any desired name)
+    - Select the Role from the dropdown 
+    - Enter a value in the Time to live text box (hover on the tooltip for more information). 
+    - Click Add.
+
+6. *API Key Created* window displays your newly created key. Make sure to copy your key and keep it secure. 
+
+### Authenticate
+
+You can authenticate your request using the HTTP header.
+
+**Example**
+
+```sh
+curl -H "Authorization: Bearer <api_key> http://127.0.0.1/graph/api/dashboards/home
```

Comment:
```suggestion
curl -H "Authorization: Bearer <api_key>" http://127.0.0.1/graph/api/dashboards/home
```

Reply:
it should work, but in our case it doesn't:
```
curl: (3) URL using bad/illegal format or missing URL
curl: (60) SSL certificate problem: self signed certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
```
so we need to pass `-k` additionally.

Reply:
my bad in example it is http - it works OK.